static/sass/initial/_type.scss

```diff
@@ -20,32 +14,28 @@
 
 
 
-// Defaults
-// --------
+// Hidden Text & Fout
+// ------------------
 
-.is-hidden {
+.is-hidden,
+.wf-loading {
   @include is-hidden;
```

Comment:
Supposedly their code is supposed to prevent FOUT without this, but I was getting a flash on every single page load — which looks real weird. In my testing, this has worked pretty well, even with my connection throttled way down, but I'd be curious about more testing.